### ISSUE TITLE
FidoMetadataDownloader::parseBlob fails with 'Unrecognized field "friendlyNames"'

### ISSUE THREAD

**Issue by georgespalding** [Yubico/java-webauthn-server#375]
Looks like the 2.5.2 version's ObjectMapper has `DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES` set to `true`.

And some time during the last day, the blob returned from https://mds.fidoalliance.org/ appears to contain fields not known to this version of the library.

The hope here is to avoid issues with this and future schema changes of the MDS format by setting `DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES` set to `false`.

**Comment by mmoayyed** [comment]
Additional details:

```
Failed to instantiate [com.yubico.webauthn.attestation.AttestationTrustSource]: Factory method 'webAuthnMetadataService' threw exception with message: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "friendlyNames" (class com.yubico.fido.metadata.MetadataStatement$MetadataStatementBuilder), not marked as ignorable (27 known properties: "tcDisplayContentType", "tcDisplayPNGCharacteristics", "alternativeDescriptions", "attestationTypes", "attachmentHint", "publicKeyAlgAndEncodings", "isKeyRestricted", "attestationCertificateKeyIdentifiers", "tcDisplay", "keyProtection", "protocolFamily", "schema", "cryptoStrength", "authenticatorVersion", "legalHeader", "aaguid", "matcherProtection", "attestationRootCertificates", "authenticatorGetInfo", "icon", "aaid", "description", "isFreshUserVerificationRequired", "userVerificationDetails", "supportedExtensions", "upv", "authenticationAlgorithms"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 549] (through reference chain: com.yubico.fido.metadata.MetadataBLOBPayload$MetadataBLOBPayloadBuilder["entries"]->java.util.HashSet[0]->com.yubico.fido.metadata.MetadataBLOBPayloadEntry$MetadataBLOBPayloadEntryBuilder["metadataStatement"]->com.yubico.fido.metadata.MetadataStatement$MetadataStatementBuilder["friendlyNames"])
```

**Comment by mmoayyed** [comment]
You may also want to consider moving this out of the relevant method and somewhere else so it's only done once:

```
ObjectMapper headerJsonMapper = JacksonCodecs.json().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, true).setBase64Variant(Base64Variants.MIME_NO_LINEFEEDS);
```

My understanding is that creating the object mapper every time a refresh op is invoked might be somewhat expensive.

**Comment by emlun** [comment]
Thank you for your patience, this is now fixed in [pre-release 2.5.3-RC2](https://github.com/Yubico/java-webauthn-server/releases/tag/2.5.3-RC2). We'll likely promote this to a stable `2.5.3` release sometime in the next few days.
